packages/wallet/src/SingleAddressWallet.ts

```diff
@@ -316,6 +316,7 @@ export class SingleAddressWallet implements ObservableWallet {
       )
       .subscribe();
 
+    // REVIEW: Any ideeas of useful debug info to log in UtxoTracker?
```

Comment:
Don't worry about this one. I've already added logs. I will push the new code soon